### PR TITLE
Accumulated Changes for Enabling LORT

### PR DESCRIPTION
**Pull request by wschin**

### CODE REVIEW

File: aten/src/ATen/templates/CompositeViewCopyKernels.cpp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #define TORCH_ASSERT_ONLY_METHOD_OPERATORS
 // ${generated_comment}
 
+#include <ATen/InferSize.h>
```
Comment (BowenBao) [inline aten/src/ATen/templates/CompositeViewCopyKernels.cpp]: Why is this needed?

File: torch/csrc/jit/ir/alias_analysis.h [hunk 2]
```diff
@@ -314,7 +314,7 @@ class AliasDb {
 // Helper check that invariants over AliasDb are maintained.
 // Useful if you are using the AliasDb mutation API and want to check you did
 // the right thing.
-void Lint(const AliasDb* db);
+TORCH_API void Lint(const AliasDb* db);
```
Comment (BowenBao) [inline torch/csrc/jit/ir/alias_analysis.h]: why is this needed?

File: torch/jit/_fuser.py [hunk 3]
```diff
@@ -48,6 +48,11 @@ def fuser(name):
         torch._C._jit_override_can_fuse_on_gpu(False)
         torch._C._jit_set_texpr_fuser_enabled(False)
         torch._C._jit_set_nvfuser_enabled(True)
+    elif name == 'none':  # Turn Pytorch fuser off
```
Comment (BowenBao) [inline torch/jit/_fuser.py:51]: Suggest separate PR into different topics: JIT side changes and exporter side changes. The benefits are
* Easier to track changes and tests.
* We fully own exporter side, so it can be reviewed and merged much faster, if without JIT changes.
* Don't need to rework on the whole thing, in case of the PR being reverted after merged.

File: torch/onnx/symbolic_opset9.py [hunk 4]
```diff
@@ -373,7 +379,10 @@ def rsub(g, self, other, alpha=None):
 
 
 def mul(g, self, other):
-    return g.op("Mul", self, other)
+    if symbolic_helper._is_bool(self):
+      return g.op("And", self, other)
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset9.py]: This looks more like an optimization step?

File: torch/onnx/symbolic_opset9.py [hunk 5]
```diff
@@ -2165,38 +2185,37 @@ def batch_norm(
...
-            "layer_norm",
-            input,
-            weight,
-            bias,
-            normalized_shape_i=normalized_shape,
-            eps_f=eps,
-            cudnn_enable_i=cudnn_enable,
-        )
+@symbolic_helper.parse_args("v", "is", "v", "v", "f")
+def native_layer_norm(g, input, normalized_shape, weight, bias, eps):
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset9.py]: Do you know where and why these `native_xxxop` are emitted? Are they LazyTensor specific?

Comment (wschin) [reply torch/onnx/symbolic_opset9.py]: It comes from both of LazyTensor and TorchDynamo.

File: torch/onnx/symbolic_opset9.py (hunk 4 above)
Comment (wschin) [reply torch/onnx/symbolic_opset9.py]: No. ONNX Mul doesn't support boolean..

File: torch/jit/_fuser.py (hunk 3 above)
Comment (wschin) [reply torch/jit/_fuser.py:51]: Sure.

File: torch/csrc/jit/ir/alias_analysis.h (hunk 2 above)
Comment (wschin) [reply torch/csrc/jit/ir/alias_analysis.h]: For writing our fusion, it's used [here ](https://github.com/microsoft/onnxruntime/blob/6fc7296b37b2776bd8b33ed632246c7594233bba/orttraining/orttraining/lazy_tensor/fusion.cpp#L401)in a [ORT PR](https://github.com/microsoft/onnxruntime/pull/10460).

File: aten/src/ATen/templates/CompositeViewCopyKernels.cpp (hunk 1 above)
Comment (wschin) [reply aten/src/ATen/templates/CompositeViewCopyKernels.cpp]: It's for resolving a LazyTensor bug after recent TorchDynamo change. It's from Pytorch team and already approved by ezyang. You can skip this commit.

**Comment by wschin** [comment]
Close as most PRs are merged into master.
